prow/config.yaml

```diff
@@ -9112,6 +9112,23 @@ periodics:
       image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
       imagePullPolicy: Always
 
+- interval: 6h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-gce-coredns-performance
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - args:
+      - --timeout=11
+      - --bare
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20180511-51cf02e75-master
+      resources:
```

Comment:
I think you can get rid of this `resources` field altogether. Defaults should likely be good enough.